aip/general/0121.md

```diff
@@ -61,6 +61,10 @@ exposes a large number of resources with a small number of methods on each
 resource. The methods can be either the standard methods ([Get][], [List][],
 [Create][], [Update][], [Delete][]), or [custom methods][].
 
+For each standard operation, a field the represents the resource **must** use
```

Comment:
"...a field **that** represents the resource..."

Reply:
ack, yes good point.

Reply:
I ended up re-writing the whole sentence. PTAL.